src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -419,7 +417,7 @@ public void ForceSomeOutOfDate()
                 DateTime time2 = File.GetLastWriteTime(t.OutputResources[1].ItemSpec);
 
                 System.Threading.Thread.Sleep(200);
-                if (NativeMethodsShared.IsOSX)
+                if (!NativeMethodsShared.IsWindows)
```

Comment:
I'm very surprised to see this. It really should be only HFS+ that's in the dark ages of one-second granularity. Are you seeing otherwise?